frontend/public/style/_overrides.scss

```diff
@@ -50,6 +51,13 @@ form.pf-v6-c-form {
   }
 }
 
+// align pf-v6-c-tabs--inset with PageSection
```

Comment:
PatternFly's `<Tabs>` does not have an `inset` value that perfectly aligns the inset with the padding of `<PageSection>`.  This is a bug?  @nicolethoen, @kmcfaul

Reply:
I'm asking about that. will report back

Reply:
I'm being told it is a bug 👍🏻 
Would you mind opening a bug in https://github.com/patternfly/patternfly/issues?
You can link to it in this override.

Reply:
[Done](https://github.com/patternfly/patternfly/issues/7376).  Will also add a link to the bug here.